Sources/NIOSSL/SSLCallbacks.swift

```diff
@@ -130,6 +130,19 @@ extension KeyLogCallbackManager {
     }
 }
 
+/// PSK Identity response type used in the callback.
+public struct PSKIdentityResponse {
```

Comment:
I think in this instance we may as well have two types, one for each callback to return. That way we don't have to deal with the question of what `identity` should be set to in the server callback.

Reply:
Addressed in 0101dcb